crm_claim_ref_smartbutton/__openerp__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+# (c) 2015 Pedro M. Baeza
+# License AGPL-3 - See LICENSE file on root folder for details
+##############################################################################
+
+{
+    "name": "Smart-button for referenced claims",
+    "version": "1.0",
+    "depends": [
+        "crm_claim",
+    ],
+    "author": "AvanzOSC, "
+              "Serv. Tecnol. Avanzados - Pedro M. Baeza, "
+              "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/crm",
+    "contributors": [
+        "Oihane Crucelaegui <oihanecrucelaegi@avanzosc.es>",
+        "Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>",
+    ],
+    "category": "Customer Relationship Management",
+    "installable": True,
+    "pre_init_hook": "pre_init_hook",
```

Comment:
You're only seeing half of the solution. This is the possible scenarios:

* You can start without the module copied in the addons path before the server starts.
* Or have it copied in addons path and start the server. This code prevents the monkey-patching: https://github.com/OCA/crm/pull/38/files#diff-df883f43bfa35a2a786daa20cc741e74R103
* When you install the module, in one or another case, the monkey-patching is done via the pre-hook script (https://github.com/OCA/crm/pull/38/files#diff-ec1e94af671fc87f2d489b43471edae2R14).
* Next time you start the server, the same line that previously prevents the monkey-patching, now does it (https://github.com/OCA/crm/pull/38/files#diff-df883f43bfa35a2a786daa20cc741e74R103).
* When you uninstall the module, the monkey-patching is undone here: https://github.com/OCA/crm/pull/38/files#diff-ec1e94af671fc87f2d489b43471edae2R20